src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -465,93 +439,79 @@ public Result addEndpointV2(@Parameter(description = "Broadcast id", required =
 									))
 	}
 			)
+
 	@POST
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Path("/{id}/rtmp-endpoint")
 	@Produces(MediaType.APPLICATION_JSON)
+	@Deprecated(since = "3.0" , forRemoval = true)
 	public Result addEndpointV3(@Parameter(description = "Broadcast id", required = true) @PathParam("id") String id,
 			@Parameter(description = "RTMP url of the endpoint that stream will be republished. If required, please encode the URL", required = true) Endpoint endpoint,
 			@Parameter(description = "Resolution height of the broadcast that is wanted to send to the RTMP endpoint. ", required = false) @QueryParam("resolutionHeight") int resolutionHeight) {
 
-		String rtmpUrl = null;
+		return addEndpointV4(id,endpoint,resolutionHeight);
+    }
+	@POST
```

Comment:
Please add documentation with openapi annodations. Keep in mind that developers uses these references. So please give the enough information to make life easy for developers.